### PR TITLE
[Bug] remove incorrect reference in testing.md

### PR DESCRIPTION
**Pull request by ananzh**
Signed-off-by: Anan Zhuang <ananzh@amazon.com>

### Description
Remove incorrect reference in testing.md
 
### Partially Resolved
https://github.com/opensearch-project/OpenSearch-Dashboards/issues/592
 
### Check List
- [ ] New functionality includes testing.
  - [x] All tests pass
- [ ] New functionality has been documented.
  - [ ] New functionality has javadoc added
- [x] Commits are signed per the DCO using --signoff

### CODE REVIEW

File: src/core/TESTING.md [hunk 1]
```diff
@@ -272,7 +272,7 @@ The tests cover:
 - authenticated / non-authenticated user access (when applicable)
 
 ```typescript
-// TODO after https://github.com/elastic/kibana/pull/53208/
```
Comment (kavilla) [inline src/core/TESTING.md:275]: Can we just remove lines 274-276 completely, and the commit message + description might be a little too broad when looking from an overview.

**Review by kavilla — COMMENTED** [review]
NIT: git stuff mentioned

but I think the empty TODO should be removed.

**Review by ananzh — COMMENTED** [review]
@kavilla I removed the whole todo concept and amend the commit. pls take another look.

**Review by kavilla — APPROVED** [review]

**Review by mihirsoni — APPROVED** [review]
LGTM !!

**Pull request merged**